src/coreclr/jit/emit.h

```diff
@@ -1650,10 +1649,33 @@ class emitter
 #endif                                     // MULTIREG_HAS_SECOND_GC_RET
     };
 
+    // TODO-Cleanup: Uses of stack-allocated instrDescs should be refactored to be unnecessary.
```

Comment:
Why should we eliminate stack-allocated instrDescs? It seems perfectly reasonable to allow them.

Reply:
We are creating these temporary `instrDesc` simply to be able to call existing methods to emit code/do printing. That seems like a code smell that the codegen/printing methods themselves should be refactored so that we can use the specific parts we want directly instead of having to indirect through an `instrDesc` structure. BTW, there's just three uses of this pattern:

* One in emitxarch.cpp that wants to call an emit method. It already has this same cleanup comment.
* One in emitarm.cpp and emitarm64.cpp for printing `LARGEJMP` instructions.